### PR TITLE
enh(centcore): Use Remote Server to send conf and external commands

### PR DESCRIPTION
**Pull request by lpinsivy**
* CLAPI enableREmote define Remote Server variable in /etc/centreon/conf.pm
* CLAPI disableREmote set Centreon variable in /etc/centreon/conf.pm
* Centcore on Central Server forward external commands to Remote Server attached to Poller if defined
*  Centcore on Central Server synchronise configuration folders with Remote Server attached to Poller if defined, and forward associated external command
* Centcore on Remote Server apply commands

### CODE REVIEW

File: lib/perl/centreon/script/centcore.pm [hunk 1]
```diff
@@ -793,8 +876,15 @@ sub initCentreonTrapd {
                                                                   logger => $self->{logger},
                                                                   timeout => 120
                                                                   );
+        } else {
+            if (defined($ns_server->{remote_id}) && $ns_server->{remote_id} != 0 && $self->{instance_mode} ne "remote") {
+            my $remote_server = $self->getServerConfig($ns_server->{remote_id});
```
Comment (cgagnaire) [inline lib/perl/centreon/script/centcore.pm]: Missing indentation until line 892.

File: src/CentreonRemote/Application/Clapi/CentreonRemoteServer.php [hunk 2]
```diff
@@ -48,6 +48,10 @@ public function enableRemote($ip)
         $result = $this->getDi()['centreon.db-manager']->getRepository(InformationsRepository::class)->authorizeMaster($ip);
         echo 'Done'. "\n";
 
+        echo "\n Set 'remote' instance type...";
+        system("sed -i -r 's/central/remote/g' " . _CENTREON_ETC_ . "/conf.pm");
```
Comment (cgagnaire) [inline src/CentreonRemote/Application/Clapi/CentreonRemoteServer.php]: Sed should be more restrictive as the word "central" can be used for password or database name or anything else.

File: src/CentreonRemote/Application/Clapi/CentreonRemoteServer.php [hunk 3]
```diff
@@ -67,6 +71,10 @@ public function disableRemote()
         $result = $this->getDi()['centreon.db-manager']->getRepository(InformationsRepository::class)->toggleRemote('no');
         echo 'Done'. "\n";
 
+        echo "\n Restore 'central' instance type...";
+        system("sed -i -r 's/remote/central/g' " . _CENTREON_ETC_ . "/conf.pm");
```
Comment (cgagnaire) [inline src/CentreonRemote/Application/Clapi/CentreonRemoteServer.php]: Sed should be more restrictive as the word "remote" can be used for password or database name or anything else.

**Comment by lpinsivy** [comment]
@cgagnaire fixes applyed

**Pull request merged**